installer/scripts/omsadmin.sh

```diff
@@ -410,7 +364,11 @@ onboard()
     echo "   <FullyQualfiedDomainName>${FQDN}</FullyQualfiedDomainName>" >> $BODY_ONBOARD
```

Comment:
can you compose the whole AgentTopologyRequest object in the Ruby? not just the Telemetry part.